### ISSUE TITLE
Fan in Quiet Mode

### ISSUE THREAD

**Issue by spyko** [RobHofmann/HomeAssistant-GreeClimateComponent#33]
**Describe the bug**
After update to 0.96+ and the newest Gree component, the Quiet Fan Mode is not as Quiet it should be. For some reason the Fan is working a little above from Low (like Medium-Low).
Although even the EWPE App shows it as Quiet, the fan is not working in Quiet Mode.
When I change from EWPE to another Fan mode and then back again to Quiet, then it is working in Quiet Speed.
However, If I do the same from Home Assistant the Quiet Mode is above the Low.

**Platform:**
 - OS: HASSIO
 - Version 0.96.3
 - Gree Component 2.0.1
 - HVAC Toyotomi Kuro

**Additional context**

**Comment by RobHofmann** [comment]
I've tried to reproduce this. When using the climate.set_fan_mode service to fan_mode: quiet, it works like intended.

Can you tell me the exact reproduction steps?

**Comment by spyko** [comment]
Today it is working as expected.
That is weird. I don't know if there is a bug in the component or in the HVAC.

However, the steps to reproduce it:
1) Set from the climate frontend the FAN to low
2) Set from the climate frontend the FAN to quiet

Instead of quiet and less air, the HVAC blowed more air.
As I said, the HA as well as EWPE app showed that the HVAC is in quiet mode but it wasn't in reality. When I changed from the EWPE to low and back in quiet then everything was ok. However, today it is working like a charm.

**Comment by RobHofmann** [comment]
If the app is showing it in quiet mode, it means that the component did succesfully deliver the message to the HVAC. My assumption is that there is little I can do from that point.

Also if you can't reproduce it and I cant either, maybe it was an one time issue.

If you dont mind i'm going to close this issue for now. Please reopen it if you happen to encounter this again. Also turn on debug logging so we have logging if this happens again.

Feel free to reopen or contact me if needed!

**Comment by spyko** [comment]
Hi,
I confront the same problem.
I turned on debug logging.
When I open the HVAC from HA, it goes to low. I suppose that it the gree component defaults to low.

Then I choose Quiet and the fan goes louder!
It is very strange behaviour.
Below the debug logs the moment I choose quiet:

2019-07-30 22:37:23 INFO (SyncWorker_2) [custom_components.gree.climate] set_fan_mode(): Quiet
2019-07-30 22:37:23 INFO (SyncWorker_2) [custom_components.gree.climate] Enabling quiet mode
2019-07-30 22:37:23 INFO (SyncWorker_2) [custom_components.gree.climate] Starting SyncState
2019-07-30 22:37:23 INFO (SyncWorker_2) [custom_components.gree.climate] Fetching(................)
2019-07-30 22:37:23 INFO (SyncWorker_2) [custom_components.gree.climate] Setting acOptions with retrieved HVAC values
2019-07-30 22:37:23 INFO (SyncWorker_2) [custom_components.gree.climate] Setting Pow: 1
2019-07-30 22:37:23 INFO (SyncWorker_2) [custom_components.gree.climate] Setting Mod: 1
2019-07-30 22:37:23 INFO (SyncWorker_2) [custom_components.gree.climate] Setting SetTem: 25
2019-07-30 22:37:23 INFO (SyncWorker_2) [custom_components.gree.climate] Setting WdSpd: 1
2019-07-30 22:37:23 INFO (SyncWorker_2) [custom_components.gree.climate] Setting Air: 0
2019-07-30 22:37:23 INFO (SyncWorker_2) [custom_components.gree.climate] Setting Blo: 0
2019-07-30 22:37:23 INFO (SyncWorker_2) [custom_components.gree.climate] Setting Health: 1
2019-07-30 22:37:23 INFO (SyncWorker_2) [custom_components.gree.climate] Setting SwhSlp: 0
2019-07-30 22:37:23 INFO (SyncWorker_2) [custom_components.gree.climate] Setting Lig: 1
2019-07-30 22:37:23 INFO (SyncWorker_2) [custom_components.gree.climate] Setting SwingLfRig: 4
2019-07-30 22:37:23 INFO (SyncWorker_2) [custom_components.gree.climate] Setting SwUpDn: 2
2019-07-30 22:37:23 INFO (SyncWorker_2) [custom_components.gree.climate] Setting Quiet: 0
2019-07-30 22:37:23 INFO (SyncWorker_2) [custom_components.gree.climate] Setting Tur: 0
2019-07-30 22:37:23 INFO (SyncWorker_2) [custom_components.gree.climate] Setting StHt: 0
2019-07-30 22:37:23 INFO (SyncWorker_2) [custom_components.gree.climate] Setting TemUn: 0
2019-07-30 22:37:23 INFO (SyncWorker_2) [custom_components.gree.climate] Setting HeatCoolType: 0
2019-07-30 22:37:23 INFO (SyncWorker_2) [custom_components.gree.climate] Setting TemRec: 0
2019-07-30 22:37:23 INFO (SyncWorker_2) [custom_components.gree.climate] Setting SvSt: 0
2019-07-30 22:37:23 INFO (SyncWorker_2) [custom_components.gree.climate] Done setting acOptions
2019-07-30 22:37:23 INFO (SyncWorker_2) [custom_components.gree.climate] Overwriting acOptions with new settings
2019-07-30 22:37:23 INFO (SyncWorker_2) [custom_components.gree.climate] Overwriting Tur: 0
2019-07-30 22:37:23 INFO (SyncWorker_2) [custom_components.gree.climate] Overwriting Quiet: 1
2019-07-30 22:37:23 INFO (SyncWorker_2) [custom_components.gree.climate] Done overwriting acOptions
2019-07-30 22:37:23 INFO (SyncWorker_2) [custom_components.gree.climate] Start sending state to HVAC
2019-07-30 22:37:23 INFO (SyncWorker_2) [custom_components.gree.climate] Done sending state to HVAC: {'t': 'res', 'mac': '.............', 'r': 200, 'opt': ['Pow', 'Mod', 'SetTem', 'WdSpd', 'Air', 'Blo', 'Health', 'SwhSlp', 'Lig', 'SwingLfRig', 'SwUpDn', 'Quiet', 'Tur', 'StHt', 'TemUn', 'HeatCoolType', 'TemRec', 'SvSt'], 'p': [1, 1, 25, 1, 0, 0, 1, 0, 1, 4, 2, 1, 0, 0, 0, 0, 0, 0], 'val': [1, 1, 25, 1, 0, 0, 1, 0, 1, 4, 2, 1, 0, 0, 0, 0, 0, 0]}
2019-07-30 22:37:23 INFO (SyncWorker_2) [custom_components.gree.climate] HA target temp set according to HVAC state to: 25
2019-07-30 22:37:23 INFO (SyncWorker_2) [custom_components.gree.climate] HA lights option set according to HVAC state to: on
2019-07-30 22:37:23 INFO (SyncWorker_2) [custom_components.gree.climate] HA health option set according to HVAC state to: on
2019-07-30 22:37:23 INFO (SyncWorker_2) [custom_components.gree.climate] HA health option set according to HVAC state to: on
2019-07-30 22:37:23 INFO (SyncWorker_2) [custom_components.gree.climate] HA powersave option set according to HVAC state to: off
2019-07-30 22:37:23 INFO (SyncWorker_2) [custom_components.gree.climate] HA sleep option set according to HVAC state to: off
2019-07-30 22:37:23 INFO (SyncWorker_2) [custom_components.gree.climate] HA operation mode set according to HVAC state to: cool
2019-07-30 22:37:23 INFO (SyncWorker_2) [custom_components.gree.climate] HA swing mode set according to HVAC state to: Fixed in the upmost position
2019-07-30 22:37:23 INFO (SyncWorker_2) [custom_components.gree.climate] HA fan mode set according to HVAC state to: Quiet
2019-07-30 22:37:23 INFO (SyncWorker_2) [custom_components.gree.climate] Finished SyncState
2019-07-30 22:37:23 INFO (MainThread) [custom_components.gree.climate] hvac_mode(): cool
2019-07-30 22:37:23 INFO (MainThread) [custom_components.gree.climate] supported_features(): 41
2019-07-30 22:37:23 INFO (MainThread) [custom_components.gree.climate] hvac_modes(): ['auto', 'cool', 'dry', 'fan_only', 'heat', 'off']
2019-07-30 22:37:23 INFO (MainThread) [custom_components.gree.climate] current_temperature(): None
2019-07-30 22:37:23 INFO (MainThread) [custom_components.gree.climate] temperature_unit(): °C
2019-07-30 22:37:23 INFO (MainThread) [custom_components.gree.climate] min_temp(): 16
2019-07-30 22:37:23 INFO (MainThread) [custom_components.gree.climate] temperature_unit(): °C
2019-07-30 22:37:23 INFO (MainThread) [custom_components.gree.climate] max_temp(): 30
2019-07-30 22:37:23 INFO (MainThread) [custom_components.gree.climate] temperature_unit(): °C
2019-07-30 22:37:23 INFO (MainThread) [custom_components.gree.climate] target_temperature_step(): 1.0
2019-07-30 22:37:23 INFO (MainThread) [custom_components.gree.climate] target_temperature_step(): 1.0
2019-07-30 22:37:23 INFO (MainThread) [custom_components.gree.climate] target_temperature(): 25
2019-07-30 22:37:23 INFO (MainThread) [custom_components.gree.climate] temperature_unit(): °C
2019-07-30 22:37:23 INFO (MainThread) [custom_components.gree.climate] fan_mode(): Quiet
2019-07-30 22:37:23 INFO (MainThread) [custom_components.gree.climate] fan_list(): ['Auto', 'low', 'Medium-Low', 'medium', 'Medium-High', 'high', 'Turbo', 'Quiet']
2019-07-30 22:37:23 INFO (MainThread) [custom_components.gree.climate] swing_mode(): Fixed in the upmost position
2019-07-30 22:37:23 INFO (MainThread) [custom_components.gree.climate] swing_modes(): ['Default', 'Swing in full range', 'Fixed in the upmost position', 'Fixed in the middle-up position', 'Fixed in the middle position', 'Fixed in the middle-low position', 'Fixed in the lowest position', 'Swing in the downmost region', 'Swing in the middle-low region', 'Swing in the middle region', 'Swing in the middle-up region', 'Swing in the upmost region']
2019-07-30 22:37:23 INFO (MainThread) [custom_components.gree.climate] name(): Hall AC
2019-07-30 22:37:23 INFO (MainThread) [custom_components.gree.climate] supported_features(): 41
2019-07-30 22:37:23 INFO (MainThread) [custom_components.gree.climate] should_poll()
2019-07-30 22:37:23 INFO (SyncWorker_4) [custom_components.gree.climate] update()
2019-07-30 22:37:23 INFO (SyncWorker_4) [custom_components.gree.climate] Starting SyncState
2019-07-30 22:37:23 INFO (SyncWorker_4) [custom_components.gree.climate] Fetching(..............)
2019-07-30 22:37:23 INFO (SyncWorker_4) [custom_components.gree.climate] Setting acOptions with retrieved HVAC values
2019-07-30 22:37:23 INFO (SyncWorker_4) [custom_components.gree.climate] Setting Pow: 1
2019-07-30 22:37:23 INFO (SyncWorker_4) [custom_components.gree.climate] Setting Mod: 1
2019-07-30 22:37:23 INFO (SyncWorker_4) [custom_components.gree.climate] Setting SetTem: 25
2019-07-30 22:37:23 INFO (SyncWorker_4) [custom_components.gree.climate] Setting WdSpd: 1
2019-07-30 22:37:23 INFO (SyncWorker_4) [custom_components.gree.climate] Setting Air: 0
2019-07-30 22:37:23 INFO (SyncWorker_4) [custom_components.gree.climate] Setting Blo: 0
2019-07-30 22:37:23 INFO (SyncWorker_4) [custom_components.gree.climate] Setting Health: 1
2019-07-30 22:37:23 INFO (SyncWorker_4) [custom_components.gree.climate] Setting SwhSlp: 0
2019-07-30 22:37:23 INFO (SyncWorker_4) [custom_components.gree.climate] Setting Lig: 1
2019-07-30 22:37:23 INFO (SyncWorker_4) [custom_components.gree.climate] Setting SwingLfRig: 4
2019-07-30 22:37:23 INFO (SyncWorker_4) [custom_components.gree.climate] Setting SwUpDn: 2
2019-07-30 22:37:23 INFO (SyncWorker_4) [custom_components.gree.climate] Setting Quiet: 1
2019-07-30 22:37:23 INFO (SyncWorker_4) [custom_components.gree.climate] Setting Tur: 0
2019-07-30 22:37:23 INFO (SyncWorker_4) [custom_components.gree.climate] Setting StHt: 0
2019-07-30 22:37:23 INFO (SyncWorker_4) [custom_components.gree.climate] Setting TemUn: 0
2019-07-30 22:37:23 INFO (SyncWorker_4) [custom_components.gree.climate] Setting HeatCoolType: 0
2019-07-30 22:37:23 INFO (SyncWorker_4) [custom_components.gree.climate] Setting TemRec: 0
2019-07-30 22:37:23 INFO (SyncWorker_4) [custom_components.gree.climate] Setting SvSt: 0
2019-07-30 22:37:23 INFO (SyncWorker_4) [custom_components.gree.climate] Done setting acOptions
2019-07-30 22:37:23 INFO (SyncWorker_4) [custom_components.gree.climate] HA target temp set according to HVAC state to: 25
2019-07-30 22:37:23 INFO (SyncWorker_4) [custom_components.gree.climate] HA lights option set according to HVAC state to: on
2019-07-30 22:37:23 INFO (SyncWorker_4) [custom_components.gree.climate] HA health option set according to HVAC state to: on
2019-07-30 22:37:23 INFO (SyncWorker_4) [custom_components.gree.climate] HA health option set according to HVAC state to: on
2019-07-30 22:37:23 INFO (SyncWorker_4) [custom_components.gree.climate] HA powersave option set according to HVAC state to: off
2019-07-30 22:37:23 INFO (SyncWorker_4) [custom_components.gree.climate] HA sleep option set according to HVAC state to: off
2019-07-30 22:37:23 INFO (SyncWorker_4) [custom_components.gree.climate] HA operation mode set according to HVAC state to: cool
2019-07-30 22:37:23 INFO (SyncWorker_4) [custom_components.gree.climate] HA swing mode set according to HVAC state to: Fixed in the upmost position
2019-07-30 22:37:23 INFO (SyncWorker_4) [custom_components.gree.climate] HA fan mode set according to HVAC state to: Quiet
2019-07-30 22:37:23 INFO (SyncWorker_4) [custom_components.gree.climate] Finished SyncState
2019-07-30 22:37:23 INFO (MainThread) [custom_components.gree.climate] hvac_mode(): cool
2019-07-30 22:37:23 INFO (MainThread) [custom_components.gree.climate] supported_features(): 41
2019-07-30 22:37:23 INFO (MainThread) [custom_components.gree.climate] hvac_modes(): ['auto', 'cool', 'dry', 'fan_only', 'heat', 'off']
2019-07-30 22:37:23 INFO (MainThread) [custom_components.gree.climate] current_temperature(): None
2019-07-30 22:37:23 INFO (MainThread) [custom_components.gree.climate] temperature_unit(): °C
2019-07-30 22:37:23 INFO (MainThread) [custom_components.gree.climate] min_temp(): 16
2019-07-30 22:37:23 INFO (MainThread) [custom_components.gree.climate] temperature_unit(): °C
2019-07-30 22:37:23 INFO (MainThread) [custom_components.gree.climate] max_temp(): 30
2019-07-30 22:37:23 INFO (MainThread) [custom_components.gree.climate] temperature_unit(): °C
2019-07-30 22:37:23 INFO (MainThread) [custom_components.gree.climate] target_temperature_step(): 1.0
2019-07-30 22:37:23 INFO (MainThread) [custom_components.gree.climate] target_temperature_step(): 1.0
2019-07-30 22:37:23 INFO (MainThread) [custom_components.gree.climate] target_temperature(): 25
2019-07-30 22:37:23 INFO (MainThread) [custom_components.gree.climate] temperature_unit(): °C
2019-07-30 22:37:23 INFO (MainThread) [custom_components.gree.climate] fan_mode(): Quiet
2019-07-30 22:37:23 INFO (MainThread) [custom_components.gree.climate] fan_list(): ['Auto', 'low', 'Medium-Low', 'medium', 'Medium-High', 'high', 'Turbo', 'Quiet']
2019-07-30 22:37:23 INFO (MainThread) [custom_components.gree.climate] swing_mode(): Fixed in the upmost position
2019-07-30 22:37:23 INFO (MainThread) [custom_components.gree.climate] swing_modes(): ['Default', 'Swing in full range', 'Fixed in the upmost position', 'Fixed in the middle-up position', 'Fixed in the middle position', 'Fixed in the middle-low position', 'Fixed in the lowest position', 'Swing in the downmost region', 'Swing in the middle-low region', 'Swing in the middle region', 'Swing in the middle-up region', 'Swing in the upmost region']
2019-07-30 22:37:23 INFO (MainThread) [custom_components.gree.climate] name(): Hall AC
2019-07-30 22:37:23 INFO (MainThread) [custom_components.gree.climate] supported_features(): 41
